packages/datetime/src/dateRangeInput.tsx

```diff
@@ -46,22 +69,82 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, {}>
         },
     };
 
+    public constructor(props: IDateRangeInputProps, context?: any) {
+        super(props, context);
+        this.state = {
+            isOpen: false,
+            selectedEnd: moment(null),
+            selectedStart: moment(null),
+        };
+    }
+
     public render() {
+        const popoverContent = (
+            <DateRangePicker
+                onChange={this.handleDateRangePickerChange}
+                maxDate={this.props.maxDate}
+                minDate={this.props.minDate}
```

Comment:
Unclear to me at this point, but I definitely think it's worth tracking as we build this thing up. But I'd rather keep the props flat-ish and have `IDateRangeInputProps` extend from `IDateRangePickerProps` if we're going to support all of the picker props.